emcc.py

```diff
@@ -2224,6 +2230,17 @@ def phase_linker_setup(options, state, newargs):
     if options.use_closure_compiler is None and settings.TRANSPILE_TO_ES5:
       diagnostics.warning('transpile', 'enabling transpilation via closure due to browser version settings.  This warning can be suppressed by passing `--closure=1` or `--closure=0` to opt into our explicitly.')
 
+  # https://caniuse.com/class: EDGE:13 FF:45 CHROME:49 SAFARI:9
+  settings.SUPPORTS_ES6_CLASSES = (settings.MIN_EDGE_VERSION >= 13 and
+                                   settings.MIN_FIREFOX_VERSION >= 45 and
+                                   settings.MIN_CHROME_VERSION >= 49 and
+                                   settings.MIN_SAFARI_VERSION >= 90000 and
+                                   settings.MIN_IE_VERSION == 0x7FFFFFFF)
+
+  if settings.EXCEPTION_STACK_TRACES and not settings.SUPPORTS_ES6_CLASSES:
```

Comment:
Is this try even for wasm exceptions?

Reply:
See: https://github.com/emscripten-core/emscripten/pull/18535#discussion_r1101218165.